### DISCUSSION TITLE
При переходе между картами в мобильной версии приложение падает в Vue2

### DISCUSSION THREAD

**Discussion by Mereibek** [yandex-maps-unofficial/vue-yandex-maps#336] [Q&A (2.x)]
Наш проект прописан на Vue 2, и мы активно используем мобильную версию.

В проекте есть две карты, каждая находится на разных страницах.
При переходе на вторую карту она падает с ошибками, ошибки указаны ниже
<img width="1342" alt="Снимок экрана 2024-07-13 в 20 53 01" src="https://github.com/user-attachments/assets/ebf694c2-4fd1-48fa-840b-ca33b89fdc26">

**Comment by daniluk4000** [comment]
Дайте, пожалуйста, репозиторий, где можно поиграться

**Reply by Mereibek** [reply]
Извини так как рабочий проект не смогу. Но попробую создать тест репо завтра

**Reply by Mereibek** [reply]
Это ошибка у нас срабатывала на старой версии 0х. Даже при обновлении карты она все еще присутствует.

В один день я переписал одну из вторых карт на нативку (первая осталось vue-yandex-maps). И вылетело ошибка offsetWidth of null.

**Reply by daniluk4000** [reply]
Что-то очень вьюшное, кажется, у вас происходит. Я гляну конечно, но я это тестировал и все было ок
